internal/distribution/distribution_test.go

```diff
@@ -56,12 +56,30 @@ func TestAvailableDistributions(t *testing.T) {
 }
 
 func TestFindPackages(t *testing.T) {
-	pkgs, err := FindPackages("../../distributions", "centos-8", "x86_64", "ssh")
+	pkgs, err := FindPackages("../../distributions", "centos-8", "x86_64", "vim")
 	require.NoError(t, err)
-	require.Greater(t, len(pkgs), 0)
-	for _, p := range pkgs {
-		require.Contains(t, p.Name, "ssh")
-	}
+	require.ElementsMatch(t, []Package{
```

Comment:
Makes sense, but explanation in the commit message would make it perfecter.

Reply:
done!